SingularityUI/app/controllers/TaskDetail.coffee

```diff
@@ -82,17 +82,36 @@ class TaskDetailController extends Controller
 
         app.showView @view
 
+        #  Refresh once again on page load to calculate initial CPU Usage value
+        @initialPageLoad = true
+        if @initialPageLoad
```

Comment:
isn't this `if` statement essentially a no-op, since it's being set to `true` in the line before? @timmfin might have some advice on how best to do the initial refresh for CPU usage data